README.md

```diff
@@ -17,6 +17,7 @@ src="https://cdn.rawgit.com/mlpack/mlpack.org/e7d36ed8/mlpack-black.svg" style="
   <a href="http://ci.mlpack.org/job/mlpack%20-%20git%20commit%20test/"><img src="https://img.shields.io/jenkins/build.svg?jobUrl=http%3A%2F%2Fci.mlpack.org%2Fjob%2Fmlpack%2520-%2520git%2520commit%2520test%2F&label=Linux%20build&style=flat-square" alt="Jenkins"></a>
   <a href="https://coveralls.io/github/mlpack/mlpack?branch=master"><img src="https://img.shields.io/coveralls/mlpack/mlpack/master.svg?style=flat-square" alt="Coveralls"></a>
   <a href="https://opensource.org/licenses/BSD-3-Clause"><img src="https://img.shields.io/badge/License-BSD%203--Clause-blue.svg?style=flat-square" alt="License"></a>
+  <a href="http://numfocus.org"><img src="https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A" alt="NumFOCUS"></a>
```

Comment:
Not sure powered by is a good wording here, also, this doesn't use the flat design as used by the other badges.

Reply:
What do you suggest instead? Sponsored by?

Reply:
If we want to follow the wording at the bottom of https://numfocus.org/project/mlpack then I guess we'd go with "Fiscally Supported Project". That seems pretty clunky though. Maybe "Sponsored Project" to match the menus? Can we write "Affiliated" without being wrong?

Reply:
I agree "Fiscally Supported Project" is a bit wordy. I'd prefer something short and sweet for the badge. 

Powered by NumFOCUS
Sponsored by NumFOCUS
Affliated to NumFOCUS (not sure if it is right or not)

These would be my picks.

Reply:
I'd vote for `Sponsored by` :+1:

Reply:
I don't really have a preference, I wonder if we really want to add that badge, I think it somewhat sticks out. However, if the mass likes the look fine with me as well.